src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,60 @@
+* {
+  box-sizing: border-box;
+}
+
+.calendar-body {
+  margin: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  height: 100vh;
+}
+
+.calendar {
+  padding: $calendar-paddings;
+  display: flex;
+  flex-wrap: wrap;
+  width: ($box-size * 7) + ($calendar-paddings * 2) + ($box-gap * 6);
+  gap: $box-gap;
+
+  &__day {
+    width: $box-size;
+    height: $box-size;
+    background-color: #EEE;
+    border: 1px solid #000;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    transition-duration: .5s;
+    font-size: 30px;
+    font-family: Arial, sans-serif;
+  }
+
+  &__day:hover {
+    cursor: pointer;
+    background-color: #FFBFCB;
+    transition-duration: .5s;
```

Comment:
list transition properties, it is better for performance

Reply:
also better move this rule to &__day and also use transition-property